setup_gen.py

```diff
@@ -25,7 +25,8 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.7',
```

Comment:
Should we also add python 3.6?

Reply:
also it does not seem as `FIX` to me as we are officially dropping support for python version and adding support for next one. more like `NEW` or even `BREAK`